x-pack/plugins/data_visualizer/public/application/common/components/top_values/top_values.tsx

```diff
@@ -70,96 +134,123 @@ export const TopValues: FC<Props> = ({ stats, fieldFormat, barColor, compressed,
+                          '+'
+                        )
                       }
-                    )}
-                    data-test-subj={`dvFieldDataTopValuesExcludeFilterButton-${value.key}-${value.key}`}
-                    style={{
-                      minHeight: 'auto',
-                      minWidth: 'auto',
-                      paddingTop: 0,
-                      paddingBottom: 0,
-                      paddingRight: 2,
-                      paddingLeft: 2,
-                    }}
+                      aria-label={i18n.translate(
+                        'xpack.dataVisualizer.dataGrid.field.addFilterAriaLabel',
+                        {
+                          defaultMessage: 'Filter for {fieldName}: "{value}"',
+                          values: { fieldName, value: value.key },
+                        }
+                      )}
+                      data-test-subj={`dvFieldDataTopValuesAddFilterButton-${value.key}-${value.key}`}
+                      style={{
+                        minHeight: 'auto',
+                        minWidth: 'auto',
+                        paddingRight: 2,
+                        paddingLeft: 2,
+                        paddingTop: 0,
+                        paddingBottom: 0,
+                      }}
+                    />
+                    <EuiButtonIcon
+                      iconSize="s"
+                      iconType="minusInCircle"
+                      onClick={() =>
+                        onAddFilter(
+                          fieldName,
+                          typeof value.key === 'number' ? value.key.toString() : value.key,
+                          '-'
+                        )
+                      }
+                      aria-label={i18n.translate(
+                        'xpack.dataVisualizer.dataGrid.field.removeFilterAriaLabel',
+                        {
+                          defaultMessage: 'Filter out {fieldName}: "{value}"',
+                          values: { fieldName, value: value.key },
+                        }
+                      )}
+                      data-test-subj={`dvFieldDataTopValuesExcludeFilterButton-${value.key}-${value.key}`}
+                      style={{
+                        minHeight: 'auto',
+                        minWidth: 'auto',
+                        paddingTop: 0,
+                        paddingBottom: 0,
+                        paddingRight: 2,
+                        paddingLeft: 2,
+                      }}
+                    />
+                  </div>
+                ) : null}
+              </EuiFlexGroup>
+            ))
+          : null}
+        {topValuesOtherCount > 0 ? (
+          <EuiFlexGroup gutterSize="xs" alignItems="center" key="other">
+            <EuiFlexItem data-test-subj="dataVisualizerFieldDataTopValueBar">
+              <EuiProgress
+                value={topValuesOtherCount}
+                max={progressBarMax}
```

Comment:
For the file data visualizer, the width of the 'Other' bar is always shorter than the other bars for me. Can that be fixed in this PR?

<img width="917" alt="image" src="https://user-images.githubusercontent.com/7405507/190434587-e7504588-2b21-4206-9c04-450de00b684e.png">

Compared to the data view after import:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/7405507/190434778-4999d957-2192-4067-8675-316a4c7eea48.png">

Reply:
Tested locally and confirmed https://github.com/elastic/kibana/pull/140667/commits/8efe008c22364edf31ac3652ddd8a910c78798a3 fixes this for me.